src/factory/ChartFactory.js

```diff
@@ -10,6 +11,20 @@ import aggregateData, { isSelectFunc } from './utils/aggregateData';
 import propTypes from '../core/propTypes';
 import withVictoryTheme from '../core/styles/withVictoryTheme';
 
+function sanitizeChartProps(chartProps, mergeProps) {
```

Comment:
We don't need this.

Reply:
@kilemensi we do, for example `parts` we need to merge it otherwise if someone passes in parts customization they will need to define everything including any default that we override.

Example i have seen we pass `props: { parts: { legendWidth: ... } }` and our defaults have parts that set axis. Without merge, we would override them.

Reply:
We could remove the Object.keys part 👍